README.md

```diff
@@ -1,80 +1,18 @@
-
-## Release lifecycle
-
-> The below is only for maintainers of the prototype kit source code - these instructions aren't relevant if you're building your own prototype using the kit.
-
-### Testing
-
-Run unit tests locally with `npm run test`.
-
-### Environments
-
-#### Preview environment
-
-Code pushed to the `main` branch will deploy on the [Preview environment](http://nhsuk-prototype-kit-preview.azurewebsites.net/).
-
-#### Live
-
-The live NHS.UK prototype kit [Live environment](https://nhsuk-prototype-kit.azurewebsites.net), accessible to the public.
-
-__Note:__ Live is behind a server side cache which results in changes not appearing immediately.
-
-### Versioning and deploying to live
-
-- Merge required changes via PR into `main` branch
-  - Ensure the version number in `package.json` and `CHANGELOG.md` match, with a summary of the changes included in the changelog
-- Verify the changes in the [Preview environment](http://nhsuk-prototype-kit-preview.azurewebsites.net/)
-- Pull down merged `main` branch to your local repository
-- Create a tag with the version number from the latest commit on `main`
-  - If the latest version you're looking to push is version "1.2.3", you can do this on the command line with
-    - `git tag v1.2.3`
-    - `git push --tags`
-- [Github actions](https://github.com/nhsuk/nhsuk-prototype-kit/actions/workflows/release.yml) will spring to life and create a release which will be visible on the [releases tab](https://github.com/nhsuk/nhsuk-prototype-kit/releases) of the repository.
-- Finally, edit the description of the release to match the content entered into the changelog.
-- To deploy the changes to Live, the promotion to Live from the Preview environment must have the appropriate approval in Azure.
-
-### Code Analysis
-
-Code analysis results can be found in [SonarQube](https://sonar.nhswebsite.nhs.uk/dashboard?id=nhsuk-prototype-kit).
+If you want to contribute to the NHS.UK prototype kit guidance, then read our [contributing guidelines](CONTRIBUTING.md).
 
 ## Support
 
-The NHS.UK prototype kit is maintained by NHS England. [Email us](mailto:service-manual@nhs.net), open a [Github issue](https://github.com/nhsuk/nhsuk-prototype-kit/issues/new) or get in touch on the [NHS digital service manual Slack workspace](https://join.slack.com/t/nhs-service-manual/shared_invite/enQtNTIyOTEyNjU3NDkyLTk4NDQ3YzkwYzk1Njk5YjAxYTI5YTVkZmUxMGQ0ZjA3NjMyM2ZkNjBlMWMxODVjZjYzNzg1ZmU4MWY1NmE2YzE).
+The NHS.UK prototype kit guidance website is maintained by NHS England. [Email us](mailto:service-manual@nhs.net), open a [Github issue](https://github.com/nhsuk/nhsuk.service-manual.prototype-kit.docs/issues/new) or get in touch on the [NHS digital service manual Slack workspace](https://join.slack.com/t/nhs-service-manual/shared_invite/enQtNTIyOTEyNjU3NDkyLTk4NDQ3YzkwYzk1Njk5YjAxYTI5YTVkZmUxMGQ0ZjA3NjMyM2ZkNjBlMWMxODVjZjYzNzg1ZmU4MWY1NmE2YzE).
```

Comment:
I think this email address has changed.

```suggestion
The NHS.UK prototype kit guidance website is maintained by NHS England. [Email us](mailto:[england.service-manual@nhs.net](mailto:england.service-manual@nhs.net)), open a [Github issue](https://github.com/nhsuk/nhsuk.service-manual.prototype-kit.docs/issues/new) or get in touch on the [NHS digital service manual Slack workspace](https://join.slack.com/t/nhs-service-manual/shared_invite/enQtNTIyOTEyNjU3NDkyLTk4NDQ3YzkwYzk1Njk5YjAxYTI5YTVkZmUxMGQ0ZjA3NjMyM2ZkNjBlMWMxODVjZjYzNzg1ZmU4MWY1NmE2YzE).
```

Reply:
Ah well spotted! We should change it in https://github.com/nhsuk/nhsuk-prototype-kit then too.